CONTRIBUTING.md

```diff
@@ -132,7 +132,7 @@ and additionally, for development we suggest to use tox and new
 versions of dependencies from pypi:
 
 ```sh
-eatmydata apt-get install -y -q python-{nose,pip,vcr,virtualenv,tox}
+eatmydata apt-get install -y -q python-{pip,vcr,virtualenv,tox}
```

Comment:
although we use pytest to run the tests, some tests still (just do `git grep nose | grep import`) use nose's functions, so nose needs to be available